Bottom: 0d82bd60b15b64c84da9811ff9293280a6eca31a
Top:    b35c5345d988f433eec8fb8e50d196d6f0f7c9c2
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2008-09-29 22:15:26 +0100

Refresh of drop-in-pcb-gl-code-various-me

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index 616dfba..bf9957c 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -1205,7 +1205,7 @@ ghid_fill_circle (hidGC gc, int cx, int cy, int radius)
   }
 }
 
-#define MAX_COMBINED_MALLOCS 1500
+#define MAX_COMBINED_MALLOCS 2500
 static void *combined_to_free [MAX_COMBINED_MALLOCS];
 static int combined_num_to_free = 0;
 
@@ -1230,7 +1230,7 @@ myFreeCombined ()
 static void
 myCombine ( GLdouble coords[3], void *vertex_data[4], GLfloat weight[4], void **dataOut )
 {
-#define MAX_COMBINED_VERTICES 1500
+#define MAX_COMBINED_VERTICES 2500
   static GLdouble combined_vertices [3 * MAX_COMBINED_VERTICES];
   static int num_combined_vertices = 0;
